f3dex2.s

```diff
@@ -667,6 +710,34 @@ f3dzex_000012BC:
     xori    $22, $22, 0x0208
     j       dma_read_write
      addi   $23, $22, -0x0158
+.else // UCODE_METHOD == METHOD_XBUS
+check_rdp_buffer_full:
+    addi $11, $23, -0xF10
+    blez $11, ovl0_04001284
+     mtc0 $23, DPC_END
+ovl0_04001260:
+    mfc0 $11, DPC_STATUS
+    andi $11, $11, 0x600
```

Comment:
This is `DPC_STATUS_END_VALID | DPC_STATUS_START_VALID`
`DPC_STATUS_END_VALID equ 0x0200` would need to be added to `rsp/rsp_defs.inc` to do this